rero_ils/modules/organisations/api.py

```diff
@@ -171,15 +171,22 @@ def get_links_to_me(self, get_pids=False):
         :param get_pids: if True list of linked pids
                          if False count of linked records
         """
-        query = LibrariesSearch()\
+        from ..acq_receipts.api import AcqReceiptsSearch
+        lib_query = LibrariesSearch()\
+            .filter('term', organisation__pid=self.pid)
+        rpt_query = AcqReceiptsSearch() \
             .filter('term', organisation__pid=self.pid)
         links = {}
         if get_pids:
-            libraries = sorted_pids(query)
+            libraries = sorted_pids(lib_query)
+            receipts = sorted_pids(rpt_query)
         else:
-            libraries = query.count()
+            libraries = lib_query.count()
+            receipts = rpt_query.count()
         if libraries:
             links['libraries'] = libraries
+        if receipts:
+            links['acq_receipts'] = receipts
```

Comment:
COSMETIC : This method isn't complete because all our resources are linked to an organisation... not only lib and receipt. But it's OK for now ! An UserStory exists concerning the data consistency and should be implements in a couple of weeks.